.github/workflows/build_conda_linux.yml

```diff
@@ -84,9 +84,7 @@ jobs:
       REF: ${{ inputs.ref }}
       CU_VERSION: ${{ matrix.desired_cuda }}
     name: ${{ matrix.build_name }}
-    runs-on: ${{ matrix.validation_runner }}
-    container:
-      image: alpine:3.14
```

Comment:
Why those two lines were deleted? It will results in slower checkout for no-op jobs

Reply:
We don't need to run-on validation_runner which maybe GPU machine

Reply:
Here is the test for:
https://github.com/pytorch/vision/actions/runs/11891587028/job/33160971728